packages/components/src/CollapsiblePanel/CollapsiblePanel.component.js

```diff
@@ -109,13 +120,25 @@ function CollapsiblePanelHeader(props) {
 		if (Array.isArray(headerItem)) {
 			const elements = headerItem.map(renderHeaderItem);
 			return (
-				<div key={index} className={classNames(css.group, css[headerColumnClass])}>
+				<div
+					key={index}
+					className={classNames(css.group, css[headerColumnClass])}
+					onClick={onToggle}
+					role="button"
+					tabIndex="0"
+				>
 					{elements}
 				</div>
 			);
 		}
 		return (
-			<div key={index} className={classNames(css[headerItem.className], css[headerColumnClass])}>
+			<div
+				key={index}
+				className={classNames(css[headerItem.className], css[headerColumnClass])}
+				onClick={onToggle}
```

Comment:
This (those) onClick are what trigger linting errors.
I've encountered this issue before, don't know yet how to solve them and it's kind of annoying ... `Talend/UI` Action maybe ?

Reply:
I agree with. Actually _div_ can be replaced with _button_ as an option